core/src/main/java/org/wildfly/extras/creaper/core/online/ExpectedFailuresBlock.java

```diff
@@ -0,0 +1,29 @@
+package org.wildfly.extras.creaper.core.online;
+
+import java.io.Closeable;
+
+/**
+ * <p>The only function this interface exposes is the ability to end a block of code started by
+ * {@link OnlineManagementClient#expectedFailures()}:</p>
+ *
+ * <pre>
+ * ExpectedFailuresBlock expectedFailures = client.expectedFailures();
+ * try {
+ *     ModelNodeResult result = client.execute(...);
+ *     ...
+ * } finally {
+ *     expectedFailures.close();
+ * }
+ * </pre>
+ *
+ * <p>If using Java 7 and above, a try-with-resources block will make the code shorter:</p>
+ *
+ * <pre>
+ * try (ExpectedFailuresBlock expectedFailures = client.expectedFailures()) {
+ *     ModelNodeResult result = client.execute(...);
+ *     ...
+ * }
+ * </pre>
+ */
+public interface ExpectedFailuresBlock extends Closeable {
+}
```

Comment:
I would suggest different name as you don't expect failures, it just can occur and you don't want the command to fail because of it.

What about naming it rather something like `AllowFailuresBlock`? WDYT?